### PR TITLE
Add funds that have left FilReserve to circ supply

### PR DESCRIPTION
**Pull request by arajasek**
Yields the following numbers at height 130000 (after the v2 actors upgrade)

FilVested 13429543.811411839397160646 FIL
FilCirculating 5961737.242508608156582553

### CODE REVIEW

**Review by magik6k — APPROVED** [review]
Looks good, but @zixuanzh should approve too

File: build/params_testground.go
Comment (zixuanzh) [inline build/params_testground.go:58]: Lets use 300_000_000 for testing too?

File: chain/stmgr/stmgr.go
Comment (zixuanzh) [inline chain/stmgr/stmgr.go:1175]: Why do we not need to add genesisPledge after Ignition epoch?

File: chain/stmgr/stmgr.go
Comment (zixuanzh) [inline chain/stmgr/stmgr.go:1186]: Let's call this `ReserveDisbursed` as a separate term.

Comment (arajasek) [reply chain/stmgr/stmgr.go]: Done

File: build/params_testground.go
Comment (arajasek) [reply build/params_testground.go]: Done

File: chain/stmgr/stmgr.go
Comment (arajasek) [reply chain/stmgr/stmgr.go:1176]: I might be wrong, but I assumed that these funds will now be accounted for in "what's missing from FilReserve", since that's where they came from originally? Is that not the case?

File: chain/stmgr/stmgr.go
Comment (zixuanzh) [inline chain/stmgr/stmgr.go:1192]: It is unclear which variable (there are two of the same name) `InitialFilReserved` refers to in build, lets make sure we are using the right precision.

**Review by zixuanzh — APPROVED** [review]
one last minor comment, lgtm

File: chain/stmgr/stmgr.go
Comment (arajasek) [reply chain/stmgr/stmgr.go:1192]: Yeah, it's using the right one (the other one will be used for testground).

**Comment by arajasek** [comment]
Good to be merged into next after #3936 lands

**Review by magik6k — APPROVED** [review]
Still looks good

**Review by magik6k — CHANGES_REQUESTED** [review]
Breaks consensus on master, rt.TotalFilCircSupply() returns >0 where it used to return 0

**Review by magik6k — APPROVED** [review]
(fixed with the last commit)

**Pull request merged**
